config/config.exs

```diff
@@ -4,4 +4,6 @@ use Mix.Config
 config :logger, :console, colors: [enabled: false]
 
 # Use higher stacktrace depth.
-config :phoenix, :stacktrace_depth, 20
\ No newline at end of file
+config :phoenix,
+  stacktrace_depth: 20,
+  filter_parameters: ["password"]
```

Comment:
I had to add this to get the test to pass, although it's already defined here: https://github.com/phoenixframework/phoenix/blob/master/mix.exs

What's the reason for that? I'm unclear on the relation between the two